src/Arrow/index.tsx

```diff
@@ -78,7 +79,7 @@ Arrow.propTypes = {
 	 * Controls how far the arrow is placed from the container's edge when
 	 * the placement contains a '-start' or '-end' suffix.
 	 */
-	distanceFromEdge: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+	distanceFromEdge: PropTypes.number,
```

Comment:
TypeScript types should replace PropTypes, can you convert the PropTypes definition to a type interface for the `Arrow` component?

You could check out the [`ThemeSection`](https://github.com/5app/base5-ui/blob/master/src/ThemeSection/index.tsx) component for an example.